docker-app/qfieldcloud/core/tests/test_api.py

```diff
@@ -26,3 +35,22 @@ def test_api_status_cache(self):
         toc = time.perf_counter()
 
         self.assertGreater(toc - tic, 0)
+
+    def test_api_default_paginator(self):
+        setup_subscription_plans()
+        user1 = Person.objects.create_user(username="user1", password="abc123")
+        token = AuthToken.objects.get_or_create(user=user1)[0]
+
+        Project.objects.create(name="project1", is_public=True, owner=user1)
+        Project.objects.create(name="project2", is_public=True, owner=user1)
+
+        items_per_page = 1
+        view = ProjectViewSet.as_view({"get": "list"})
+        request = APIRequestFactory().get(
+            "/api/v1/projects/", {"items_per_page": items_per_page}
+        )
+        force_authenticate(request, user=user1, token=token)
+        response = view(request)
+        response_rendered = response.render()
+        results = response_rendered.data["results"]
+        self.assertEqual(len(results), items_per_page)
```

Comment:
Can you also verify that the current way of working (with no limit) is still returning the correct results. As far as I understand your PR, you are only testing when the new get parameter exists.